tests/integration/test_sqs.py

```diff
@@ -520,6 +542,9 @@ def test_receive_terminate_visibility_timeout(self, sqs_client, sqs_queue):
         result = sqs_client.receive_message(QueueUrl=queue_url)
         assert "Messages" not in result
 
+    def test_update_message_visibility_timeout(self):
+        pass
+
```

Comment:
Is this for the other PR, #6573 ?

Reply:
oh yea, actually started working on that and then it spiraled into this PR ;-) must have left it here